micromind/networks/yolo.py

```diff
@@ -406,13 +424,28 @@ def forward(self, p3, p4, p5):
         h1 = self.up2(x)
         h1 = torch.cat((h1, p3), dim=1)
         head_1 = self.n2(h1)
-        h2 = self.n3(head_1)
-        h2 = torch.cat((h2, x), dim=1)
-        head_2 = self.n4(h2)
-        h3 = self.n5(head_2)
-        h3 = torch.cat((h3, p5), dim=1)
-        head_3 = self.n6(h3)
-        return [head_1, head_2, head_3]
+        return_heads = []
+        """
```

Comment:
same typos as above. also, maybe this can be replaced by in-line comments after the specific "if-else" statements..
something like "# here we check if the first head is used" and so on..